tests/scaletest/lib/ingress.go

```diff
@@ -456,6 +458,15 @@ func DeletePod(podName string, namespace string) error {
 	return nil
 }
 
+func WaitForAKOPodReboot(t *testing.T, akoPodName string) {
+	t.Logf("Waiting for AKO pod...")
+	pod, _ := coreV1Client.Pods(AVISYSTEM).Get(ctx, akoPodName, metaV1.GetOptions{})
+	if pod.Status.Phase != coreV1.PodRunning {
+		time.Sleep(5 * time.Second)
```

Comment:
If WaitForAKOPodReboot is already waiting for the Pod, then we don't need the sleep here.

Reply:
As this is a recursive function, sleep has been added here to limit the number of recursive calls.

Reply:
I missed that. In that case may be we can use g.Eventually() like other test cases here.